modules/ROOT/pages/enable-inbound-traffic-self.adoc

```diff
@@ -1,3 +1,35 @@
-= Enable Inbound Traffic with a Domain
+= Configure Domains and Inbound Traffic (RTF Self-managed Kuberenetes)
+
+To enable external application to connect to Runtime Fabric on self-managed Kubernetes you must provide an Ingress controler that is compatible with your Kubernetes environment. Runtime fabric uses the Kubernetes controller to handle routing and load balancing functionality. 
+
+Inbound traffic is enabled by default. This allows Mule applications or API gateways to listen on inbound connections. You can turn off inbound traffic per application when deploying or configuring an application. You can disable enbound traffic per application in Runtime Manager when deploying or configuring an application.
+
+Runtime Fabric allows you to configure domains that serve as endpoints for your deployed applications.
+
+== Configure Domains for 
+
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Select the name of your Runtime Fabric to open the management page.
+. Select *Inbound Traffic*, then click *Add Endpoints.*
+. 
+
+
+. [NEED ADDITIONAL INFO]
+
+== Verify that Your Application Is Accesible 
```

Comment:
I think the procedure in this section will need to be updated since we don't have a controller IP in this mode.

Reply:
Remove for now